models/aether_lat-lon/readme.rst

```diff
@@ -0,0 +1,191 @@
+Aether Rectangular Grid Interface
+=================================
+
+Overview
+--------
+
+The `Aether`_ ("eether") space weather model can be implemented 
+on a logically rectangular grid "lat-lon", 
+or on an the cubed-sphere grid (see ../aether_cubed_shere).
+This is the interface to the lat-lon version.
+
+.. _Aether: https://aetherdocumentation.readthedocs.io/en/latest/
+
+Aether writes history and restart files, with some overlap of the fields (?).
+The restart fields are divided among 2 types of files: neutrals and ions.
+They are further divided into "blocks", which are subdomains of the globe.
+Blocks start in the southwest corner of the lat/lon grid and go east first, 
+then to the west end of the next row north and end in the northeast corner. 
+Each block has a halo around it filled with field values from neighboring blocks.
+All of these need to be combined to make a single state vector for filter.
+There's a unique set of these files for each member.
+The restart file names reflect this information:  
+
+|   {neutrals,ions}_mMMMM_gBBBB.nc
+|   MMMM = ensemble member (0-based)
+|   BBBB = block number (0-based)
+
+The restart files do not have grid information in them, which must be read from
+   grid_gBBBB.nc
+
+Aether_to_dart and dart_to_aether read the same namelist; transform_state_nml.
+The fields chosen to be part of the model state are specified in ``variables``.
+Program aether_to_dart will read the specified fields, from all the restarts
```

Comment:
link to aether_to_dart and dart_to_aether docs in this paragraph

Reply:
@johnsonbk and @kdraeder  settled on a single readme for this model.
Should the aether_to_dart and dart_to_aether docs be split out, and point to those new rst files,
or should this point to the source code for the programs, or ...?

Reply:
my bad, no links needed.